lib/platform/common.ts

```diff
@@ -34,7 +34,7 @@ export interface RepoResult {
 
 export interface RepoParams {
   localDir: string;
-  optimizeForDisabled: boolean;
+  optimizeForDisabled?: boolean;
```

Comment:
Defaults to `false`, so should be okay. When it's optional, we may not care about this field in `platform/*` code.

Reply:
Do we use it anymore in platform though?

Reply:
It's now used here:

https://github.com/renovatebot/renovate/blob/e82e63351e6588ce8c9f065c9d68113ae0d29d94/lib/workers/repository/init/apis.ts#L20